bittensor/_subtensor/subtensor_impl.py

```diff
@@ -364,6 +382,11 @@ def kappa (self, netuid: int, block: Optional[int] = None ) -> Optional[int]:
     def difficulty (self, netuid: int, block: Optional[int] = None ) -> Optional[int]:
         if not self.subnet_exists( netuid ): return None
         return self.query_subtensor( "Difficulty", block, [netuid] ).value
+    
+    """ Returns network Burn hyper parameter """
+    def burn (self, netuid: int, block: Optional[int] = None ) -> Optional[bittensor.Balance]:
```

Comment:
Can we change this naming with something more specific? We might have more parameters later that also require burning